tests/unit/src/test/scala/tests/markdown/NestSuite.scala

```diff
@@ -171,8 +185,38 @@ class NestSuite extends BaseMarkdownSuite {
        |""".stripMargin
   )
 
+  check(
```

Comment:
One of the tests where it has to be split, as implicit values in Scala 3 have to have an explicit type